webextensions/manifest/permissions.json

```diff
@@ -509,9 +509,9 @@
             }
           }
         },
-        "menu": {
+        "menus": {
           "__compat": {
-            "description": "<code>menu</code>",
+            "description": "<code>menus</code>",
```

Comment:
This line is unnecessary as the renderer defaults to <code>\`&lt;code&gt;${name}&lt;/code&gt;\`</code>.